open-metadata-implementation/access-services/analytics-modeling/analytics-modeling-api/src/main/java/org/odpi/openmetadata/accessservices/analyticsmodeling/model/module/Table.java

```diff
@@ -8,6 +8,9 @@
  *	Abstraction of the table in MOSER module.
```

Comment:
What is MOSER?

Reply:
Modeling Server web application in Cognos Analytics